packages/theme/modules/catalog/pages/product.vue

```diff
@@ -560,9 +572,48 @@ export default defineComponent({
 
       fetchProducts();
     };
+
+    const addToCart = async () => {
+      await addItem({ product: product.value, quantity: qty.value });
+
+      if (
+        cameFromWishlistPage.value
+        && isProductInWishlist.value
+        && !cartError.value?.addItem
+      ) {
+        await removeItemFromWishlist({ product: product.value });
+
+        sendNotification({
+          id: Symbol('product_added_to_cart_from_wishlist_product_page'),
+          message: i18n.t(`You added ${getProductName(
+            product.value,
+          )} to your shopping cart.`) as string,
```

Comment:
Please pass product name as parameter to the label. We can't possibly make thousands of translation labels for each product name

https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting

```suggestion
          message: i18n.t(`You added productname to your shopping cart.`, { productname: getProductName(product.value)} ) as string,
```

This won't render the product name after this change obviously, so you'll need to add that to the translations in en.js, de.js like `'You added productname to your shopping cart': 'You added {productname} to your shopping cart'`.

Don't forget to do that in MyWishlist aswell.

@bartoszherba is it ok to add new translation labels like that?

Reply:
No, the product's translation comes from Magento. We mustn't translate it on the front. Good catch, thanks.